docs/routers.md

```diff
@@ -0,0 +1,50 @@
+## Routers
```

Comment:
Empty line after each title please .. also start at highest level (#) and dont skip levels